### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#498] [Esami & Esoneri M-Z]
### Traccia

<img width="538" alt="image" src="https://github.com/user-attachments/assets/8a32c6a6-8e6a-4b81-8f4a-afe50fb03b42">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/df07f71c-d881-4245-bc2c-c7480205b298)

![image](https://github.com/user-attachments/assets/2f90565f-26a3-46de-b902-99eae24ab68d)
